virttest/utils_test/libvirt.py

```diff
@@ -2232,6 +2233,24 @@ def create_channel_xml(params, alias=False, address=False):
     return channelxml
 
 
+def add_panic_device(vm_name, model='isa', addr_type='isa', addr_iobase='0x505'):
+    """
+    Create panic device xml
+
+    :param vm_name: name of vm
+    :param model: panic model
+    :param addr_type: address type
+    :param addr_iobase: address iobase
+    """
+    vmxml = vm_xml.VMXML.new_from_dumpxml(vm_name)
```

Comment:
what will happen if there is a panic device already present in xml, do you want to handle that logic aswell?

Reply:
Fixed. Thanks